src/test/suite/workspace.test.ts

```diff
@@ -1,24 +1,21 @@
 'use strict'
 
+import * as path from 'path'
```

Comment:
I believe most of these `* as x` should just be `import path from 'path'` — the stars are not needed, they work in some libraries, but not all

Reply:
What you’re saying it true for ESM, but not for TypeScript that’s compiled to CJS, which is what this is.